### PR TITLE
Feature home page

### PR DESCRIPTION
**Pull request by kamilsdz**
A bit of content on the home page:)

### CODE REVIEW

File: app/views/pages/home.html.haml [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+  %p.lead= t('.create_service_book')
+  %p= t('.log_in_now')
   %p.lead
-    %a.btn.btn-primary.btn-lg{ href: "#", role: "button" } Learn more
+    %a.btn.btn-primary.btn-lg{ href: new_user_session_path, role: 'button' }
+      = t('.log_in')
+%h4.text-center= t('.why_is_worth')
+%h5= t('.we_have_several_reasons')
+.list-group
+  %a.list-group-item.list-group-item-action{ href: '#' }= t('.less_failures')
```
Comment (fab-ian) [inline app/views/pages/home.html.haml]: Unused ```<a>``` tag. Add some link, or remove this tag.

File: app/views/pages/home.html.haml [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+  %a.list-group-item.list-group-item-action{ href: '#' }= t('.increase_value')
+  %a.list-group-item.list-group-item-action{ href: '#' }= t('.makes_you_happy')
+= image_tag 'home/motohelper.jpg', class: 'rounded mx-auto d-block'
```
Comment (fab-ian) [inline app/views/pages/home.html.haml]: Great idea to add motohelper printscreen, but please add picture without OS information. Neutral solution. And maybe a little more information ;)

Make the image responsible.
https://getbootstrap.com/docs/4.0/content/images/#responsive-images

File: app/views/pages/home.html.haml [hunk 3]
```diff
@@ -8,10 +8,11 @@
 %h4.text-center= t('.why_is_worth')
 %h5= t('.we_have_several_reasons')
 .list-group
-  %a.list-group-item.list-group-item-action{ href: '#' }= t('.less_failures')
-  %a.list-group-item.list-group-item-action{ href: '#' }= t('.increase_value')
-  %a.list-group-item.list-group-item-action{ href: '#' }= t('.makes_you_happy')
-= image_tag 'home/motohelper.jpg', class: 'rounded mx-auto d-block'
+  %button.list-group-item.list-group-item-action{ type: 'button' }= t('.less_failures')
+  %button.list-group-item.list-group-item-action{ type: 'button' }= t('.increase_value')
+  %button.list-group-item.list-group-item-action{ type: 'button' }= t('.makes_you_happy')
```
Comment (fab-ian) [inline app/views/pages/home.html.haml]: Why do you use buttons? Buttons should attached to an action :) These are only information. The 'hover' behavior is confusing.

Comment (kamilsdz) [reply app/views/pages/home.html.haml]: For me, it's just more dynamically. But I'm not front-end developer or UI/UX guy, so maybe you have right. I'll improve it.

**Pull request merged**